### DISCUSSION TITLE
Scrawler.exe?

### DISCUSSION THREAD

**Discussion by CynicalRose** [AAndyProgram/SCrawler#50] [Q&A] [answered]
I looked at the FAQ says to double click a Scrawler.exe but non to be found.  I see a SLN file do I do something with that.  Still new to all this stuff.

**Comment by AAndyProgram** [comment, marked as answer]
https://github.com/AAndyProgram/SCrawler/releases/latest

**Comment by mariusgerome** [comment]
I found the .exe file in the latest release of yours.
I'm on Mac. If I click on the .exe file, nothing happens.
Which Program do I use, to open .exe files – the console? I tried it.
Do I need to move the latest.zip inside the other folder, with the scrawler.sln and extract it there?

**Reply by AAndyProgram** [reply]
Windows only
https://github.com/AAndyProgram/SCrawler#requirements
